xrayutilities/materials/cif.py

```diff
@@ -197,6 +211,11 @@ def floatconv(string):
                 # add a comma to a fraction to avoid int division problems
                 opstr = re.sub(r"/([1-9])", r"/\1.", opstr)
                 self.symops.append(opstr)
+            elif oxidation_loop: # atom oxidation state
+                loop_start = False
+                asplit = line.split()
+                oxstate = asplit[-1]
+                self.oxstates.append(oxstate)
```

Comment:
is the `oxstates` variable used anywhere else? what are the motivations for this changes? Can you submit a cif file which needs these changes to be parsed correctly?

Reply:
Not needed.
Main motivation for `oxstates` was to handle changes in oxidation states during electronic phase transitions for resonant reflectivity simulations... but should be handled differently (i.e. load new crystal file, or change scattering factors).